packages/icons/src/lib/icons/index.ts

```diff
@@ -115,6 +120,7 @@ export { default as Stars } from './Stars';
 export { default as SushiSwap } from './SushiSwap';
 export { default as Telegram } from './Telegram';
 export { default as Testnet } from './Testnet';
+export { default as TokenWhite } from './TokenWhite';
 export { default as Token } from './Token';
```

Comment:
Looks like this was missed in an earlier PR.
Could you change the new icon's name to -> `Token`
And the old one to `TokenColored` 
renaming the files and running `pnpm icons:create` should do the trick 😄 

reason -> This new icon is not really white it can take any color using `fill`. And old one cannot do that it has hardcoded colors.